src/ert/resources/forward-models/res/script/ecl_config.py

```diff
@@ -203,8 +205,33 @@ class FlowConfig(EclConfig):
 
     def __init__(self):
         config_file = os.getenv("FLOW_SITE_CONFIG", default=self.DEFAULT_CONFIG_FILE)
+        if not Path(config_file).exists():
+            config_file = self.init_flow_config()
         super().__init__(config_file, simulator_name="flow")
 
+    @staticmethod
+    def locate_flow_binary() -> str:
+        """Locate the path for a flow executable.
+
+        Returns the empty string if there is nothing to be found in $PATH."""
+        foundpath = shutil.which("flow")
+        if foundpath is not None:
+            return foundpath
+        return ""
+
+    @staticmethod
+    def init_flow_config() -> str:
+        binary_path = FlowConfig.locate_flow_binary()
```

Comment:
If the binary path comes out as `""`, will the rest of the execution not lead to some weird error down the line?

Reply:
This would lead to "Executable not defined" down the line, but I agree that might be a weird error a bit hard to understand what is going on. 

I will throw an error .